README.md

```diff
@@ -106,6 +106,26 @@ const output = {
 }
 ```
 
+## Usage with TypeScript
+You can use TypeScript definition from [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/inline-style-prefixer) using [@types/inline-style-prefixer](https://www.npmjs.com/package/@types/inline-style-prefixer)
+
+```sh
+npm install --save @types/inline-style-prefixer
+```
+
+Then import in your code:
+
+```typescript
+import prefixAll = require('inline-style-prefixer/static');
```

Comment:
It would be pure JavaScript (using global require function). `import` tells TypeScript to look for definitions according to set module system (AMD, commonjs) and make static analysis with that definitions...